### PR TITLE
rancher/net:v0.8.2, rancher/network-manager:v0.4.1, MTU fixes

### PR DESCRIPTION
**Pull request by leodotcloud**

### CODE REVIEW

**Comment by deniseschannon** [comment]
@leodotcloud it's hard to follow exactly what was changed when the first commit combines updates and the copy from the previous version. is it possible to update the commits to separate them? 1 commit for the base copy and the following commits for your changes.

File: infra-templates/ipsec/2/docker-compose.yml [hunk 1]
```diff
@@ -6,16 +6,22 @@ services:
     command: sh -c "echo Refer to router sidekick for logs; mkfifo f; exec cat f"
     network_mode: ipsec
     ports:
-    - 500:500/udp
-    - 4500:4500/udp
+      - 500:500/udp
+      - 4500:4500/udp
+    # Force cni-driver to start first
+    depends_on:
```
Comment (deniseschannon) [inline infra-templates/ipsec/2/docker-compose.yml]: i'm not sure if we should rely/use `depends_on` as once we launch we don't export this value. 

@alena1108 is there another way to say which services should start first?

**Comment by leodotcloud** [comment]
@deniseschannon @alena1108 switched back to `volumes_from`

**Comment by deniseschannon** [comment]
LGTM
